neuro_san/service/http/server/http_server_app.py

```diff
@@ -137,10 +147,23 @@ def get_stats(self) -> str:
         return str(stats_dict)
 
     def log_request(self, handler):
-        request = handler.request
-        metadata: Dict[str, Any] = handler.get_metadata()
-        status = handler.get_status()
-        duration = 1000 * request.request_time()  # in milliseconds
-        # handler.logger is our custom HttpLogger
-        handler.logger.info(metadata, "%d %s %s (%s) %.2fms",
-                            status, request.method, request.uri, request.remote_ip, duration)
+        if isinstance(handler, BaseRequestHandler):
+            request = handler.request
+            metadata: Dict[str, Any] = handler.get_metadata()
+            status = handler.get_status()
+            duration = 1000 * request.request_time()  # in milliseconds
+            # handler.logger is our custom HttpLogger
+            handler.logger.info(metadata, "%d %s %s (%s) %.2fms",
+                                status, request.method, request.uri, request.remote_ip, duration)
+        elif isinstance(handler, ErrorHandler):
```

Comment:
Use our custom logging logic for BaseRequestHandler (and all our subclassed handlers) only.